pyiron/sphinx/base.py

```diff
@@ -743,7 +743,6 @@ def restart(
         )
 
         new_job.input = self.input
-        new_job.load_default_groups()
```

Comment:
Not sure about this.  It was just recently added in 12a530d2a / a35d483b1 I guess with the intention to fill in input fields that are not touched by user input.  For restarting a calculation `self.input` might already be populated with defaults from the original run, though

Reply:
Yes, I remember to have added this line for some reason, but removing it does not seem to cause problems so I removed it, since conceptually `restart` for me is equivalent to `create_job` and therefore should not do `load_default_groups()` which `create_job` doesn't do.

Reply:
Ah, alright, that works then.